From 14bcb6a28d2a8fb49dfc7b700309ccc758bd7039 Mon Sep 17 00:00:00 2001
From: Haithem Salah <haithem.salah@intel.com>
Date: Tue, 5 Apr 2016 14:40:58 +0200
Subject: [PATCH 14/41] Remove Silead Power ON/OFF sequence as it introduces
 touch issues on many devices (touch loss after boot)

Per L MR1 code line, this sequence is not needed, as during resumse
the TS driver is getting reset.

Change-Id: I2eb0cc1ee27387e424916ba547688628d93f8e8b
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-31785
Signed-off-by: Haithem Salah <haithem.salah@intel.com>
Reviewed-on: https://android.intel.com:443/490610
Reviewed-on: https://android.intel.com:443/505229
---
 drivers/input/touchscreen/silead.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/input/touchscreen/silead.c b/drivers/input/touchscreen/silead.c
index 52198fe..43e47c4 100644
--- a/drivers/input/touchscreen/silead.c
+++ b/drivers/input/touchscreen/silead.c
@@ -415,10 +415,6 @@ static int silead_ts_setup(struct i2c_client *client)
 	int ret;
 	u32 status;
 
-	silead_ts_set_power(client, SILEAD_POWER_OFF);
-	msleep(20);
-	silead_ts_set_power(client, SILEAD_POWER_ON);
-	msleep(20);
 
 	ret = silead_ts_get_id(client);
 	if (ret)
-- 
1.9.1

